pkg/apis/metering/v1alpha1/report.go

```diff
@@ -116,6 +116,7 @@ type ReportScheduleMonthly struct {
 type ReportStatus struct {
 	Conditions     []ReportCondition `json:"conditions,omitempty"`
 	LastReportTime *meta.Time        `json:"lastReportTime,omitempty"`
+	NextReportTime *meta.Time        `json:"nextReportTime,omitempty"`
```

Comment:
what sets this?

Reply:
https://github.com/operator-framework/operator-metering/pull/610/files#diff-d249f334a4e9a4b17d52d7b1f95289faR497

Github suppressed the diff to `reports.go` so you probably need to expand it.